cmd/release-controller/main.go

```diff
@@ -402,7 +405,26 @@ func (o *options) Run() error {
 		initializeMetrics(bugzillaErrorMetrics)
 		c.bugzillaErrorMetrics = bugzillaErrorMetrics
 	}
-	klog.V(4).Infof("8: %v", time.Now().Sub(start))
+
+	if o.VerifyJira {
+		ghClient, err := o.github.GitHubClient(false)
+		if err != nil {
+			return fmt.Errorf("Failed to create github client: %v", err)
+		}
+		ghClient.Throttle(o.githubThrottle, 0)
```

Comment:
We probably only need to initialize a single github client that can be shared by both Bugzilla and Jira.

Reply:
Fixed. Github client is initialised separately.